### ISSUE TITLE
Import all extensions

### ISSUE THREAD

**Issue by golf1052** [golf1052/code-sync#3]
I don't know if I can just copy over the entirety of an extension folder from one place to another place and have it just work.

From the documentation [here](https://code.visualstudio.com/docs/extensions/install-extension#_sharing-privately-with-others-sideloading) it looks like I should just be able to copy the folder over.

However [this](https://code.visualstudio.com/docs/extensions/patterns-and-principles#_using-nodejs-modules-with-extensions) makes me think that just copying over extensions will not work for every single extension.

Basically I just need to test importing entire extensions to see if it works correctly.

**Comment by golf1052** [comment]
Just did a small test (OS X 10.10 using OneDrive, seems to be working. Still need to do more testing to since I ran into some weird issues. Code is in https://github.com/golf1052/code-sync/tree/sync-all

**Comment by golf1052** [comment]
Extension has been rewritten so this can be closed.
